README.md

```diff
@@ -1,6 +1,6 @@
-[![Build status](https://github.com/firewalld/firewalld/workflows/testsuite/badge.svg)](https://github.com/firewalld/firewalld/actions/workflows/testsuite.yml?query=event%3Apush+workflow%3Atestsuite)
-[![Integration status](https://github.com/firewalld/firewalld/workflows/integration%20testsuite/badge.svg)](https://github.com/firewalld/firewalld/actions?query=event%3Apush+branch%3Amaster+workflow%3Aintegration-testsuite)
```

Comment:
I was reading about the badges and AFAICS if you omit the `branch=` it will use the "default branch", i.e. main - or whatever we change it to in the future.

What do you think?

Reply:
We could emit the branch from the badge icon, but removing it from the workflow URL would show the release results too.

Reply:
Yes. Make sense to me. @thaller agree?

Reply:
I agree and have removed the branch from the badges.